src/Core/Checkout/Customer/Event/CustomerAddressUpsertedEvent.php

```diff
@@ -0,0 +1,40 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Checkout\Customer\Event;
+
+use Shopware\Core\Checkout\Customer\Aggregate\CustomerAddress\CustomerAddressEntity;
+use Shopware\Core\Checkout\Customer\CustomerEntity;
+use Shopware\Core\Framework\Context;
+use Shopware\Core\Framework\Event\ShopwareEvent;
+use Symfony\Contracts\EventDispatcher\Event;
+
+class CustomerAddressUpsertedEvent extends Event implements ShopwareEvent
```

Comment:
should implement `ShopwareSalesChannelEvent`

```suggestion
class CustomerAddressUpsertedEvent extends Event implements ShopwareSalesChannelEvent
```